app/upgrades/v8/upgrades.go

```diff
@@ -27,7 +27,6 @@ func setInitialMinCommissionRate(ctx sdk.Context, keepers *keepers.AppKeepers) {
 			v.Commission.Rate = stakingParams.MinCommissionRate
 			v.Commission.UpdateTime = ctx.BlockHeader().Time
 
-			v.Commission.UpdateTime = v.Commission.UpdateTime.AddDate(0, 0, -2) // TODO: remove this after testing
```

Comment:
When you have time, can you please think about our testing approach there, on how this could be achieved without modifying the code?

Basically, how our upgrade testing pipeline should be modified to allow injection of such parameters.